xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -1251,8 +1251,8 @@ namespace PVR
     if (!CServiceBroker::GetPVRManager().IsStarted() || IsRunningChannelScan())
       return false;
 
-    PVR_CLIENT scanClient;
-    std::vector<PVR_CLIENT> possibleScanClients = CServiceBroker::GetPVRManager().Clients()->GetClientsSupportingChannelScan();
+    CPVRClientPtr scanClient;
+    std::vector<CPVRClientPtr> possibleScanClients = CServiceBroker::GetPVRManager().Clients()->GetClientsSupportingChannelScan();
```

Comment:
Does any addon actually support channel scanning?

Reply:
No, not at all, it's just a few afaik. But this is not related to this PR. Or did I break something here?

Reply:
No, I just thought of it when I saw it here